bin/container_daemon

```diff
@@ -1,16 +1,7 @@
 #!/bin/sh
 set -e
-user=ipfs
 repo="$IPFS_PATH"
 
-if [ `id -u` -eq 0 ]; then
-  # ensure folder is writable
-  su-exec "$user" test -w "$repo" || chown -R -- "$user" "$repo"
```

Comment:
This is needed to fix permissions (#3744) if you run `docker run -v /does/not/exist/yet:/data/ipfs go-ipfs`, otherwise go-ipfs can't write to it's home and crashes. This is also the reason why there's no `USER ipfs` in the dockerfile itself, as the permissions need to be preserved and then dropped in the entrypoint. :)